provider.tf

```diff
@@ -0,0 +1,11 @@
+provider "aws" {
+  alias = "acm_account"
+}
+
+provider "aws" {
+  alias = "route53_account"
+}
+
+// Do not remove, this causes input prompts otherwise
+// known terraform bug
+provider "aws" {}
```

Comment:
Can you please add a reference to the issue so that we can remove it in the event that it gets fixed upstream?